backend/src/main/kotlin/com/tngtech/apicenter/backend/connector/rest/dto/SpecificationFileDto.kt

```diff
@@ -1,5 +1,13 @@
 package com.tngtech.apicenter.backend.connector.rest.dto
+
 import com.fasterxml.jackson.annotation.JsonCreator
+import io.swagger.annotations.ApiModel
+import io.swagger.annotations.ApiModelProperty
 import java.util.UUID
 
-data class SpecificationFileDto @JsonCreator constructor(val fileContent: String?, val fileUrl: String? = "", val id: UUID? = null)
\ No newline at end of file
+@ApiModel(description = "Represents an OpenAPI-File")
+data class SpecificationFileDto @JsonCreator constructor(
+    @ApiModelProperty("The content of a file to be uploaded - either this or fileUrl has to be set.") val fileContent: String?,
+    @ApiModelProperty("The url of a remote file to be loaded - either this or fileContent has to be set.") val fileUrl: String? = "",
+    @ApiModelProperty("The uuid of an already existing specification (optional)") val id: UUID? = null
```

Comment:
`@ApiModelProperty` has a property `required`, so that the "(optional)" is not needed.
What I'm wondering is whether we can somehow tell the spec that either `fileContent` or `fileUrl` is needed. If we make `SpecificationFileDto` an interface with two implementations, one for `fileContent` and one for `fileUrl`, do you think the deserialization will still work? That could be a bit cleaner.